entrypoint.sh

```diff
@@ -18,6 +18,14 @@ sh -c "git config --global credential.username $GITLAB_USERNAME"
 sh -c "git config --global core.askPass /cred-helper.sh"
 sh -c "git config --global credential.helper cache"
 sh -c "git remote add mirror $*"
+if [[ -n "${REMOVE_BRANCH}" ]] && [[ "${REMOVE_BRANCH}" == "true" ]]; then # Check if variable exists and is true
+   # If branch exists; w/o sh -c ""
+   branchExists=$(git ls-remote $(git remote get-url --push mirror) ${CHECKOUT_BRANCH:-$DEFAULT_GITHUB_REF} | wc -l)
+   if [[ "${branchExists}" == "1" ]]; then
+      echo "removing the branch ${branch} at $(git remote get-url --push mirror)"
```

Comment:
Can you just flip the `branch` and `${branch}` parts. Sorry, I missed that there was a similar line at the end of the file.